core/controller/src/main/scala/whisk/core/controller/Actions.scala

```diff
@@ -214,11 +214,11 @@ trait WhiskActionsApi extends WhiskCollectionAPI {
      * - 409 Conflict
      * - 500 Internal Server Error
      */
-    override def create(user: Identity, name: EntityName)(implicit transid: TransactionId) = {
```

Comment:
tip: since this commit reverts the previous, for future reference and to help in reviewing, you can squash it into the previous commit (or cherry pick and drop the previous commit as needed).